src/client.rs

```diff
@@ -13,7 +13,7 @@ pub use signatures::{SignatureError, Verification};
 use crate::client::signatures::openssl_verifier::OpenSSLVerifier as DefaultVerifier;
```

Comment:
Having a type and an import alias with the same name is a little confusing but, more importantly, is hard for grepping the code.  Maybe `ActiveVerifier`?